tests/Branding/LogoAssetTest.php

```diff
@@ -0,0 +1,109 @@
+<?php
+declare(strict_types = 1);
+namespace Tests\App\Branding;
+
+// this test works with the local asset files in /assets (not S3 or web/assets)
+
+use Tests\App\BaseWebTestCase;
+
+class LogoAssetTest extends BaseWebTestCase
+{
+    protected $logos = null;
+    const ASSET_PATH = 'assets/images/logos';
```

Comment:
```suggestion
    private const ASSET_PATH = 'assets/images/logos';
```
Code sniffer says that visibility must be declared after PHP 7.1
I don't think this matter in a test but not hurting to have it